gui/webdisplay/src/TWebWindowsManager.cxx

```diff
@@ -159,6 +157,37 @@ void ROOT::Experimental::TWebWindowsManager::Unregister(ROOT::Experimental::TWeb
    //   }
 }
 
+//////////////////////////////////////////////////////////////////////////
+/// Provide URL address to access specified window from inside or from remote
+
+std::string ROOT::Experimental::TWebWindowsManager::GetUrl(ROOT::Experimental::TWebWindow &win, bool remote)
+{
+   if (!fServer) {
+      R__ERROR_HERE("GetUrl") << "Server instance not exists";
```

Comment:
The string argument to `R__ERROR_HERE` is actually a "package", e.g. maybe "WebDisplay" in your case. It allows to tweak the suppression level differently, for different parts of the code.